chapter3/challenges/program/lib/password_manager_2.rb

```diff
@@ -1,2 +1,54 @@
 class PasswordManager2
-end
\ No newline at end of file
+    def initialize
+        #Initializes three arrays, password manager being the main one, and two other arrays for checking / ease of ordering.
+        @password_manager = Array.new()
+        @password_array = Array.new()
+        @service_array = Array.new()
+    end
+        def add(service, password)
```

Comment:
I appreciate you had to copy and paste this back into the branch, but the indentation is now incorrect in a few places. I use Rufo ruby formatter to do get VSCode to do it for me. Maybe we can discuss that.

Reply:
I'll take a look now!